R/eval.R

```diff
@@ -58,8 +58,7 @@ evaluate <- function(input,
 
   parsed <- parse_all(input, filename, stop_on_error != 2L)
   if (inherits(err <- attr(parsed, 'PARSE_ERROR'), 'error')) {
-    source <- new_source(parsed$src)
-    output_handler$source(source)
+    source <- new_source(parsed$src, call[[1]], output_handler$source)
```

Comment:
`call[[1]]` is not the right object here. Should probably be `parsed$expr[[1]]` which will be `expression()` as it is an error

Reply:
Hmmmm, since the source code isn't parseable, we should emit some dummy output that indicates that. I think `expression()` is probably reasonable since there's no other way you could get that.